planning/behavior_velocity_traffic_light_module/src/scene.cpp

```diff
@@ -356,9 +358,9 @@ bool TrafficLightModule::findValidTrafficSignal(TrafficSignalStamped & valid_tra
   const auto traffic_signal_stamped_opt = planner_data_->getTrafficSignal(
     traffic_light_reg_elem_.id(), true /* traffic light module keeps last observation */);
   if (!traffic_signal_stamped_opt) {
-    RCLCPP_WARN_THROTTLE(
-      logger_, *clock_, 5000 /* ms */,
-      "the traffic signal data associated with regulatory element id is not received");
+    RCLCPP_WARN_STREAM_ONCE(
+      logger_, "the traffic signal data associated with regulatory element id "
+                 << traffic_light_reg_elem_.id() << " is not received");
```

Comment:
@shmpwk @tkimura4 @satoshi-ota are you alright with making this warn once? (Instead of every 5 seconds)

This is to reduce clutter.

Reply:
I agree to warn once either.